### PR TITLE
Add ability to set DHW operating mode 

### PR DESCRIPTION
**Pull request by pavelsidla**
Opearting mode set ability to (eco,comfort,off)

### CODE REVIEW

**Comment by pavelsidla** [comment]
Hello can you add this feature, i need to set different operating mode in day schedule.
FG: 
I am going to bath around 18:00-22:00, so i need to change operating mode to comfort because i wont to wait till water will be heated.

File: PyViCare/PyViCareHeatingDevice.py [hunk 1]
```diff
@@ -196,6 +196,11 @@ def setDomesticHotWaterTemperature2(self, temperature):
         return self.service.setProperty("heating.dhw.temperature.temp2", "setTargetTemperature",
                                         {"temperature": int(temperature)})
 
+    @handleAPICommandErrors
+    def setDhwOperatingMode(self, mode):
```
Comment (crazyfx1) [inline PyViCare/PyViCareHeatingDevice.py]:
```suggestion
    def setDomesticHotWaterOperatingMode(self, mode):
```

Comment (crazyfx1) [reply PyViCare/PyViCareHeatingDevice.py]: Could you please change the method name to be consistent with the other method names?

**Review by crazyfx1 — CHANGES_REQUESTED** [review]

File: PyViCare/PyViCareHeatingDevice.py (hunk 1 above)
Comment (pavelsidla) [reply PyViCare/PyViCareHeatingDevice.py]: Hi its done :)

**Review by crazyfx1 — APPROVED** [review]

**Pull request merged**